packages/graphql/src/services/stacks.ts

```diff
@@ -2617,256 +2617,362 @@ export async function respondToQuestion(
   },
```

Comment:
Did not actually change much in this file beside adding comment for future logging and moving the entire thing inside try/catch (to be able to log the error later). could undo this change if we want to keep it assessment only for now